site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-Changelog-11-January-2022.adoc

```diff
@@ -0,0 +1,114 @@
+= Apache Cassandra Changelog #11
+:page-layout: single-post
+:page-role: blog-post
+:page-post-date: January 18, 2022
+:page-post-author: The Apache Cassandra Community
+:description: The Apache Cassandra Community
+:keywords: 
+
+image::blog/changelog_header.jpg[Apache Cassandra Changelog]
+Our monthly roundup of key activities and knowledge to keep the community informed.
+
+== Release Notes
+
+=== Released
+
+Available since 7th September, the latest release of Apache Cassandra is  https://www.apache.org/dyn/closer.lua/cassandra/4.0.1[4.0.1,window=_blank], (https://archive.apache.org/dist/cassandra/4.0.1/apache-cassandra-4.0.1-bin.tar.gz.asc[pgp^],  https://archive.apache.org/dist/cassandra/4.0.1/apache-cassandra-4.0.1-bin.tar.gz.sha256[sha256^], and https://archive.apache.org/dist/cassandra/4.0.1/apache-cassandra-4.0.1-bin.tar.gz.sha512[ sha512^]). This was a rapid release to fix a https://issues.apache.org/jira/browse/CASSANDRA-16877[critical bug^] in Gossip on large clusters. Please read the https://gitbox.apache.org/repos/asf?p=cassandra.git;a=blob_plain;f=CHANGES.txt;hb=refs/tags/cassandra-4.0.1[release notes^] and let us know if you encounter any problems.
+
+Note: As the docs are not yet updated, the bintray location for Debian users is replaced with the https://apache.jfrog.io/artifactory/cassandra/[ASF's JFrog Artifactory location^].
+
+See the https://cassandra.apache.org/download/[download section^] for the latest stable and older supported versions of source and binary distributions.
+
+To stay up-to-date, we recommend joining the Cassandra  https://cassandra.apache.org/community/[mailing list^].
+
+== Community Notes
+
+_Updates on Cassandra Enhancement Proposals (CEPs), how to contribute, and other community activities._ 
+
+_Are you new to the project? We have established a https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=484[New Release tracking Kanboard^] and a https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=484&quickFilter=2162&quickFilter=2160[“Starter Tickets”^] quick label that corresponds to our Low Hanging Fruit status. Any of these tickets should be of appropriate complexity for someone new to the project to tackle. You can also reach out in the #cassandra-dev Slack channel on the-asf.slack.com and ask for assistance from any of the 13 designated Cassandra mentors!
```

Comment:
I'd like to suggest adding a link to https://cassandra.apache.org/_/community.html so new users can find out how to get access to ASF Slack. 👍